package.json

```diff
@@ -3,6 +3,9 @@
   "description": "Accessibility engine for automated Web UI testing",
   "version": "3.0.0-alpha.4",
   "license": "MPL-2.0",
+  "engines": {
+    "node": ">=4"
```

Comment:
Do we need to specify a node version if we're downgrading it to use `var`?

Reply:
it just produces a warning for anybody running on older versions, so it's useful